ui/app/router.js

```diff
@@ -17,6 +17,15 @@ Router.map(function () {
       this.route('dashboard');
       this.mount('config-ui');
       this.mount('sync');
+      // TODO remove conditional once further feature work for single item recovery for release 1.21 is completed
+      if (config.environment !== 'production') {
```

Comment:
The router is a special place in ember land. You might want to double check that nothing weird happens once you build this for prod.